wagtail/wagtaildocs/templates/wagtaildocs/chooser/chooser.js

```diff
@@ -57,6 +58,13 @@ function(modal) {
             dataType: 'text',
             success: function(response){
                 modal.loadResponseText(response);
+            },
+            error: function(response, textStatus, errorThrown) {
+                {% trans "Report this error to your webmaster with the following information:" as server_error %}
```

Comment:
Hello, @nfletton!

We should be careful with translated string: `trans` may return string that can break JS code. It can be string with `'` character for example.

I think it would be good to apply `escapejs` filter with all non-constant strings.